djangosaml2/views.py

```diff
@@ -48,6 +48,7 @@
 from saml2.ident import code, decode
 from saml2.sigver import MissingKey
 from saml2.s_utils import UnsupportedBinding
+from saml2.request import AuthnRequest
```

Comment:
This doesn't seem like the right `AuthnRequest`... `pysaml2` declares _two_ different classes with that name:
https://github.com/IdentityPython/pysaml2/blob/master/src/saml2/samlp.py#L1224
https://github.com/IdentityPython/pysaml2/blob/master/src/saml2/request.py#L157

...and, when reproducing https://github.com/knaperek/djangosaml2/issues/168 locally, which this PR seems to aim to fix, I'm seeing `request_xml` appear as the `samlp` variety and not the `request` variety:

```python
ipdb> request_xml
<saml2.samlp.AuthnRequest object at 0x10f74b780>
ipdb> AuthnRequest
<class 'saml2.request.AuthnRequest'>
ipdb> isinstance(request_xml, AuthnRequest)
False
```

Reply:
> This doesn't seem like the right `AuthnRequest`... `pysaml2` declares _two_ different classes with that name:
> https://github.com/IdentityPython/pysaml2/blob/master/src/saml2/samlp.py#L1224
> https://github.com/IdentityPython/pysaml2/blob/master/src/saml2/request.py#L157
> 
> ...and, when reproducing #168 locally, which this PR seems to aim to fix, I'm seeing `request_xml` appear as the `samlp` variety and not the `request` variety:
> 
> ```python
> ipdb> request_xml
> <saml2.samlp.AuthnRequest object at 0x10f74b780>
> ipdb> AuthnRequest
> <class 'saml2.request.AuthnRequest'>
> ipdb> isinstance(request_xml, AuthnRequest)
> False
> ```

:man_facepalming: thanks, good catch.